app/forms/InternationalAddressForm.scala

```diff
@@ -84,15 +84,21 @@ class InternationalAddressForm @Inject()(configConnector: ConfigConnector) {
           country => country.flatMap(_.name)
             .getOrElse(throw new InternalServerException("Country missing from international address"))
         )
-    )(Address.apply(_, _, _, _, _, _, _))(address => Address.unapply(address).map {
+    )
+    ((ln1, ln2, ln3, ln4, ln5, postcode, country) => Address.apply(ln1, Some(ln2), ln3, ln4, ln5, postcode, country))
+    (address => Address.unapply(address).map {
       case (l1, optL2, optL3, optL4, optL5, optPostcode, optCountry, _) =>
-        (l1, optL2, optL3, optL4, optL5, optPostcode, optCountry)
+        (
+          l1,
+          optL2.getOrElse(throw new InternalServerException("[InternationalAddressForm] Missing address line 2")),
```

Comment:
Probably better to make it get or else "", that wouldn't give an unexplained error to users and they'd still need to submit line 2 to get through the page.